src/frontend/src/tests/test-support/test-data/authenticated-user.stub.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * This file is part of NER's FinishLine and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { RoleEnum, AuthenticatedUser } from 'shared';
+
+export const exampleAuthenticatedAdminUser: AuthenticatedUser = {
+  userId: 2,
+  firstName: 'Joe',
+  lastName: 'Shmoe',
+  email: 'shmoe.j@husky.neu.edu',
+  emailId: 'shmoe.j',
+  role: RoleEnum.ADMIN,
+  favoritedProjectsId: [],
+  changeRequestsToReviewId: []
+};
+
+export const exampleAllAuthenticatedUsers: AuthenticatedUser[] = [exampleAuthenticatedAdminUser];
```

Comment:
why do you need this

Reply:
@Peyton-McKee b/c the res function in test-utils.stub.ts line 90 requires AutehnticatedUser as an input. Previously, AuthenticatedUser had the same type with User but now it requires favoritedProjectsId and changeRequestsToReviewId fields to be defined. 

This is why I decided to add a new file, authenticated-user.stub.ts to define the example of authenticated user.

Reply:
Alternatively, I realized we can also use exampleAdminUser by adding favoritedProjectsId and changeRequestsToReviewId fields through spread syntax. 

-> {...exampleAdminUser, favoritedProjectsId: [], changeRequestsToReviewId: []} in test-utils.stub.ts, so we can delete authenticated-user.stub.ts if we want to.

Reply:
Lmk your thoughts on this. 

I need to push anyways to delete the comment in test-utils.stub.ts